src/clients/FtpClient.cc

```diff
@@ -563,17 +618,24 @@ Ftp::Client::handleEpsvReply(Ip::Address &remoteAddr)
     /*  ANSI sez [^0-9] is undefined, it breaks on Watcom cc */
     debugs(9, 5, "scanning: " << ctrl.last_reply);
 
-    buf = ctrl.last_reply + strcspn(ctrl.last_reply, "(");
+    const auto line = ctrl.last_reply;
+    const auto open = strchr(line, '(');
+    if (!open) {
+        debugs(9, DBG_IMPORTANT, "ERROR: Missing EPSV reply from " <<
```

Comment:
This is a transfer protocol failover state, it happens and is not usually fixable by admin. So bump it down to the protocol detailed debug where it will not be annoying, but leave the prefix to highlight for anyone wanting to check the behaviour.
```suggestion
        debugs(9, 2, "ERROR: Missing EPSV reply from " <<
```